packages/markdown-docx/src/OoxmlTransformer.js

```diff
@@ -151,6 +151,17 @@ class OoxmlTransformer {
                 $class: `${NS_PREFIX_CommonMarkModel}Softbreak`,
             };
         case 'w:r':
+            if(element.elements[0].name==='w:rPr'){
+                let emphFound = element.elements[0].elements.some(subElement=>{
+                    return subElement.name==='w:i' && subElement.attributes['w:val'] === 'true';
+                });
+                if(emphFound){
+                    return {
+                        $class: `${NS_PREFIX_CommonMarkModel}Emph`,
+                        nodes:[...this.deserializeElements(element.elements)]
+                    };
+                }
+            }
```

Comment:
I will see to this later in the morning.

Reply:
Improve the spacing by using a good formatter such as prettier. You can integrate its extension in VSCode. Make sure the rules your configure are _almost_ consistent with the entire codebase.